freqtrade/rpc/external_message_consumer.py

```diff
@@ -270,6 +270,11 @@ async def _receive_messages(
                     logger.debug(f"Connection to {channel} still alive...")
 
                     continue
+                except (websockets.exceptions.ConnectionClosed):
+                    # Just eat the error and continue reconnecting
+                    logger.warning(f"Disconnection in {channel} - retrying in {self.sleep_time}s")
+                    await asyncio.sleep(self.sleep_time)
```

Comment:
are you sure this will actually try to reconnect?
we're not "really" calling `.connect()` again - only `wait_for()` ...

Reply:
Yes it works just like the ping error does, by breaking out of that loop and continuing the outer loop on `_create_connection`.